cpp/oneapi/dal/algo/pca/backend/cpu/infer_kernel.cpp

```diff
@@ -0,0 +1,97 @@
+/*******************************************************************************
+* Copyright 2020 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include <daal/src/algorithms/pca/transform/pca_transform_kernel.h>
+
+#include "oneapi/dal/algo/pca/backend/cpu/infer_kernel.hpp"
+#include "oneapi/dal/backend/interop/common.hpp"
+#include "oneapi/dal/backend/interop/error_converter.hpp"
+#include "oneapi/dal/backend/interop/table_conversion.hpp"
+#include "oneapi/dal/algo/pca/infer_types.hpp"
```

Comment:
Don't need to include `infer_types.hpp`, it's already included by `cpu/infer_kernel.hpp`